### PR TITLE
Change ChainSpec::from_genesis for Rialto and Millau chains to reflect the chain names.

### LINKED ISSUE #1078: Change ChainSpec::from_genesis for Rialto and Millau chains to reflect the chain names.
At the moment Rialto and Millau are being created with "Local Testnet" name and this should be changed to retrieve the corresponding names as Rococo or Polkadot.

This change is needed so the bridge UI can get the name and get the bridgesId dynamically.
https://github.com/paritytech/parity-bridges-ui/blob/master/src/util/getConfigs.ts#L32

### PR DESCRIPTION
**Pull request by hbulgarini**
Closes #1078

### CODE REVIEW

File: bin/millau/node/src/chain_spec.rs [hunk 1]
```diff
@@ -81,8 +81,8 @@ impl Alternative {
 		);
 		match self {
 			Alternative::Development => ChainSpec::from_genesis(
-				"Development",
-				"dev",
+				"Millau",
+				"millau",
```
Comment (hbulgarini) [inline bin/millau/node/src/chain_spec.rs:85]: I figured out that i need to change the names as well for Development otherwise UI would not work with development nodes. In case this is wrong i would just revert this change.

**Comment by hbulgarini** [comment]
cc @tomusdrw

**Review by svyatonik — APPROVED** [review]
Probably "Millaur Deverlopment"/"Millau_dev" for development chainspec?

File: bin/millau/node/src/chain_spec.rs (hunk 1 above)
Comment (adoerr) [reply bin/millau/node/src/chain_spec.rs:85]: I agree with @svyatonik, there should be some way to distinguish the development chain spec from the local test net one. Something like;
- `Millau Development` / `millau_dev`

and

- `Millau Local` / `millau_local`

**Review by adoerr — APPROVED** [review]

**Comment by hbulgarini** [comment]
@svyatonik @adoerr , names changed as suggested. Thanks!

**Review by adoerr — APPROVED** [review]
